sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_blobstoragecsaio.py

```diff
@@ -237,7 +237,7 @@ async def update_checkpoint(self, checkpoint: Dict[str, Any]) -> None:
     async def list_checkpoints(
         self, fully_qualified_namespace, eventhub_name, consumer_group
     ):
-        blob_prefix = "{}/{}/{}/checkpoint".format(
+        blob_prefix = "{}/{}/{}/checkpoint/".format(
```

Comment:
Is this going to break the checkpointing for anyones existing checkpoint store?

Reply:
nope, it won't break anything, all the checkpoint/ownership blob path are of the format
"<namespace>/<eventhub>/<consumer group>/checkpoint/<partition_id>"

Reply:
Ah of course - this is just the search prefix. Ignore me :)